resources/js/profile-page/cover.tsx

```diff
@@ -126,9 +126,9 @@ export default class Cover extends React.Component<Props> {
     return (
       <>
         {this.props.user.is_supporter &&
-          <span className='profile-info__icon profile-info__icon--supporter' title={trans('users.show.is_supporter')}>
-            {times(this.props.user.support_level ?? 0, (i) => <span key={i} className='fas fa-heart' />)}
-          </span>
+          <a href={route('support-the-game')} className='profile-info__icon profile-info__icon--supporter' title={trans('users.show.is_supporter')}>
```

Comment:
there's reason I mentioned adjust style accordingly in the previous comment

![](https://s.myconan.tk/2023/10/firefox_2023-10-31_15-19-17.png)